weather.py

```diff
@@ -57,8 +60,14 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
-
+    with open(csv_file) as csv_file:
+        reader = csv.reader(csv_file)
+        data = list(reader)[1:]
+        result = []
+        for row in data:
+            if len(row) > 0:
+                result.append([row[0], int(row[1]), int(row[2])])
```

Comment:
Nice application of string slicing and casting reader to a List. This works fine, though the alternative is to use `next(reader)` to skip the header. `next(reader)` might be preferred when we start working with larger files, because doing both (typecasting the whole file to a List + then looping through it) can be expensive compared to looping through it once (i.e. `for row in reader`)